pkg/api/serialization_test.go

```diff
@@ -73,6 +78,27 @@ var apiObjectFuzzer = fuzz.New().NilChance(.5).NumElements(1, 1).Funcs(
 		c.Fuzz(&j.Selector)
 		j.Replicas = int(c.RandUint64())
 	},
+	// Roles and RoleBindings maps and slices are never nil
+	func(j *authorizationv1.Policy, c fuzz.Continue) {
+		if j.Roles == nil {
+			j.Roles = make([]authorizationv1.NamedRole, 0, 0)
+		}
+	},
+	func(j *authorizationapi.Policy, c fuzz.Continue) {
+		if j.Roles == nil {
```

Comment:
> I don't think you have to check for nil here... you'll always get the default fuzzed object

No.

--- FAIL: TestTypes (0.34 seconds)
	serialization_test.go:185: About to test api.DeploymentConfig
	serialization_test.go:185: About to test api.BuildConfig
	serialization_test.go:185: About to test api.PolicyBindingList
panic: runtime error: assignment to entry in nil map [recovered]
	panic: runtime error: assignment to entry in nil map

goroutine 22 [running]:
runtime.panic(0x852400, 0xb39ed3)